float32 start_angle                             #The angle of the first beam wrt the front beam
float32 angle_increment 
float32 range_min
float32 range_max
float32[] ranges_front
float32[] ranges_rear
Header  header
geometry_msgs/PoseWithCovariance pose             #The pose of the robot e.g. position:[x,y,theta] , orientation: in quaternion domain 

